patroni/postgresql.py

```diff
@@ -1367,16 +1369,16 @@ def save_configuration_files(self):
         try:
             for f in self._configuration_to_save:
                 if os.path.isfile(f):
-                    shutil.copy(f, f + '.backup')
+                    shutil.copy(f, f.replace(self._config_dir, self._data_dir) + '.backup')
```

Comment:
You're right. I'm not really satisfied of this replace too. I will follow your advice and push a new branch as soon as possible. Thanks.